doc/source/commands/flight/cooked.rst

```diff
@@ -157,35 +157,41 @@ Like all ``LOCK`` expressions, the steering and throttle continually update on t
 
 .. warning::
 
-    You cannot ``WAIT`` during the execution of the expression in a
+    It's a very bad idea to ``WAIT`` during the execution of the expression in a
     LOCK WHEELSTEERING.  See the note in the next section below.
 
 
-Cannot 'WAIT' during cooked control calculation
------------------------------------------------
+Don't 'WAIT' during cooked control calculation
+----------------------------------------------
 
 Be aware that because LOCK THROTTLE, LOCK STEERING, LOCK
-WHEELTHROTTLE, and LOCK WHEELSTEERING cause your expression
+WHEELTHROTTLE, and LOCK WHEELSTEERING are actually
+:ref:`triggers <triggers>` that cause your expression
 to be calculated every single physics update tick behind
-the scenes, you cannot execute a ``WAIT`` command during
-the evaluation of the value used in them.
+the scenes, you should not execute a ``WAIT`` command
+in the code that performs the evaluation of the value
+used in them, as that will effectively cheat the entire
+script out of the full execution speed it deserves.
 
 For example, if you attempt this::
 
     function get_throttle {
-	wait 1.  // this line won't work.
-	return 0.5.
+        wait 0.001.  // this line is a bad idea.
+        return 0.5.
     }
     lock throttle to get_throttle().
 
-Then the ``WAIT`` command won't work.  You can't make the
-system pause execution while it's trying to run the
-expression that tells it what to do 25 times a second.
-
-The entire expression that you LOCK any of the four cooked
-controls to must execute and finish quickly enough that it can
-be called 25 times a second (on typical default game settings
-that's how often the steering manager will run your expression).
+Then the ``WAIT`` command will be getting hit *every single*
```

Comment:
Should we mention that this will not technically happen every single update, that it will actually delay the evaluation of both the trigger and main line code, and may cause the evaluation to be out of sync with the current conditions?